src/twisted/test/test_ssl.py

```diff
@@ -667,8 +671,10 @@ def test_method(self):
         # SSLv23_METHOD allows SSLv2, SSLv3, or TLSv1
         self.assertEqual(self.context._method, SSL.SSLv23_METHOD)
 
-        # And OP_NO_SSLv2 disables the SSLv2 support.
-        self.assertTrue(self.context._options & SSL.OP_NO_SSLv2)
+        # OpenSSL >= 1.1.0 no longer provides SSLv2
+        if not OPENSSL_1_1:
```

Comment:
Hm, I thought it was clearer to call out the OpenSSL version in code, but a comment will do.  I'll switch the asserts to check that the value's in the options as @alex suggested.